### DISCUSSION TITLE
Why is there no [skip ci] or [ci skip] keyword in commit messages after 2017?

### DISCUSSION THREAD

**Discussion by ayberkyasa** [twbs/bootstrap#37693] [Q&A]
Hi folks!

I was using GitHub Search API for a research study and realized that [skip ci] or [ci skip] keyword, which can make the workflow runs skipped for the commit whose message includes one of these keywords, suddenly disappear after 2017. I checked some versions of the "Contributing Guidelines" with the hope to find some rules about that. However, I could not find anything. 

By the way, you can see [skip ci] or [ci skip] keyword's behavior from the following GitHub documentation link: https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs

Is there anyone who might know the reason for this sudden disappearance? I think there should be a reason to change the developer's behavior about that.

**Comment by ayberkyasa** [comment]
@XhmikosR or @cvrebert  Could you please provide information to me?

**Reply by mdo** [reply]
> Is there anyone who might know the reason for this sudden disappearance? I think there should be a reason to change the developer's behavior about that.

We just stopped doing it. 🤷🏻‍♂️

**Reply by ayberkyasa** [reply]
> > Is there anyone who might know the reason for this sudden disappearance? I think there should be a reason to change the developer's behavior about that.
> 
> We just stopped doing it. 🤷🏻‍♂️

Is there any reason that you just stopped doing it? I would appreciate it if you can provide more information to me.

**Reply by julien-deramond** [reply]
A part of the answer (which is only my own interpretation) is that for commits merged into the main branch, it is safer to let the workflow do its work to avoid any regression. Moreover, our CI has special rules depending on the context (_main_ branch, PRs, etc.). That's why probably you'll find nothing by checking the history of, let's say, the _main_ branch.
In PRs, IMO it's because nobody thinks about it even if it would be better in some cases in terms of eco-design.
